contracts/SaleVesting.sol

```diff
@@ -112,8 +122,15 @@ contract SaleVesting is Ownable {
                 _vestingDetails[i].claimStartTime
             );
 
+            totalVestingAmount +=
```

Comment:
what's this for?
if you want totalVestingAmount, it should be 
totalVestingAmount + =_vestingDetails[i].vestingAmount;